pants.travis-ci.toml

```diff
@@ -8,6 +8,10 @@ execution_strategy = "subprocess"
 # overhead or else just generally thrash the container and slow things down.
 travis_parallelism = 4
 
+[GLOBAL]
+# Override color support detection - we want colors and Travis supports them.
+use_colors = true
```

Comment:
The name of this option is just `colors`: https://github.com/pantsbuild/pants/blob/07cb7d72f8939628711bc2923ce721a704913c5b/src/python/pants/option/global_options.py#L901-L908

So:
```
colors = true
```